spec/requests/api/v1/articles_spec.rb

```diff
@@ -1,8 +1,9 @@
 require "rails_helper"
+require "swagger_helper"
```

Comment:
We need to require `swagger_helper` in any API tests with rswag DSL in order for documentation generation to run correctly